src/main/kotlin/pl/allegro/mobile/logic/operators/numeric/GreaterOrEqualOperation.kt

```diff
@@ -5,7 +5,7 @@ import pl.allegro.mobile.logic.NumberElement
 import pl.allegro.mobile.logic.ClientLogicMarker
 import pl.allegro.mobile.logic.operators.OperatorFactory
 
-internal interface GreaterOrEqualThanOperation {
+internal interface GreaterOrEqualOperation {
```

Comment:
Shouldnt it be Greater**Than**OrEqual**To**/Less**Than**OrEqual**To**? :D

Reply:
For me it's to verbose :)